tests/integration/test_smoke.py

```diff
@@ -108,13 +108,27 @@
 ]
 
 
+# These services require a particular region to run.
+REGION_OVERRIDES = {
+    'route53domains': 'us-east-1'
+}
+
+
+def _aws(command_string):
+    service = command_string.split()[0]
+    command = command_string
+    if service in REGION_OVERRIDES:
+        command += ' --region %s' % REGION_OVERRIDES[service]
+    return aws(command)
```

Comment:
aws() takes `env_vars` as a param.  why not just provide it there?

Reply:
It doesn't make that much of a difference, but I can see where there could be some issues depending on how the smoke test is written. I'll get that changed.